Runtime/GameLiftAnywhereCore.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using AmazonGameLiftPlugin.Core.CredentialManagement;
+using AmazonGameLiftPlugin.Core.CredentialManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared.FileSystem;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace AmazonGameLift.Runtime
+{
+    public sealed class GameLiftAnywhereCore
```

Comment:
This class can be merged with https://github.com/aws/amazon-gamelift-plugin-unity/blob/c068e1dc2ab1e454713ee3c8c6ae0e2077b6a60f/Runtime/Core/ApiGatewayManagement/AmazonGameLiftClientWrapper.cs#L11

Reply:
`AmazonGameLiftClientWrapper` is essentially a wrapper over the gamelift’s AWS client to be used for invoking GameLift APIs. It should be where we make all GameLift APIs calls.
Arbitrarily separating the APIs by functions “connectionAPIs”, “anywhereAPIs”, “matchmakingAPIs” will make it difficult to maintain later